src/AggregateFunctions/AggregateFunctionDistinct.h

```diff
@@ -236,6 +236,11 @@ class AggregateFunctionDistinct : public IAggregateFunctionDataHelper<Data, Aggr
         return true;
     }
 
+    bool isState() const override
```

Comment:
Ok. It means we can have both `aggDistinctState` and `aggStateDistinct` that will act similar?

Reply:
Yes.